src/test/routes/find-choice.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from 'chai';
```

Comment:
we will need some unit tests most likely too making sure the content is displayed along with starting some e2e tests

Reply:
maybe the e2e isnt part of this story, maybe it is, we should have a discussion with @johnoneill46 to figure out when to play those. it would stem from making sure the `Start Now` button on the home page successfully takes you to this page